test/version_commands.bats

```diff
@@ -352,9 +352,9 @@ teardown() {
 }
 
 @test "global should preserve symlinks when updating versions" {
-  mkdir $HOME/other-dir
-  touch $HOME/other-dir/.tool-versions
-  ln -s other-dir/.tool-versions $HOME/.tool-versions
+  mkdir "$HOME/other-dir"
+  touch "$HOME/other-dir/.tool-verions"
```

Comment:
Typo. I would have expected this test to fail with this typo. Can you investigate?

Reply:
The touch isn't needed because at least in Bash, when writing to a symlink that points to a non-existent location, that location is automatically created. That is also why the tests didn't fail on the previous revision, when `.tool-versions` wasn't the basename. And since the touch isn't needed, nothing happens when it creates the wrong file.

Reply:
Ah okay, thanks